src/components/SearchResultDetail/SearchResultDetail.tsx

```
@@ -60,6 +60,9 @@ const SearchResultDetail = ({ id }: { id: string }) => {
           <p className="detail-search-result__info">
             Total number of non ad words: {searchResult?.nonAdLinksCount || 0}
           </p>
+          <p className="detail-search-result__info">
+            Search Engine: {searchResult?.searchEngine || ''}
```

There is no case when this can be null, or something is very wrong with the application. So in the latter case, it is best to let the application crash/error out.